src/bmsg.c

```diff
@@ -309,9 +309,10 @@ int main (int argc, char *argv [])
             puts ("  publish eula <state>");
             puts ("                         publish <state> of EULA on stream " FTY_PROTO_STREAM_EULA );
             puts ("                         Currently the only state used by the system is ACCEPTED.");
-            puts ("  publish licensing-limitation <value> <item> <category> [<value> <item> <category> ...]");
-            puts ("                         publish <value> <item> <category> on stream " FTY_PROTO_STREAM_LICENSING_ANNOUNCEMENTS);
-            puts ("                         See description details in licensing agent for valid items and categories.");
+// disabled for release
```

Comment:
I believe a cleaner solution for comparisons is some `ifdef` or at least a block comment, rather than an horde of `//` prefixes :)

Can this be a configure or runtime option (to still have this in development-env builds)?

At the very least, I'd ask to consider these points if we decide to do this change in `master` as well.

Reply:
devel Vs deploy may be (part of) the answer

Reply:
good point for master, definitely bmsg will need some love, as for this release let's keep it as is.